scripts/deploy-and-test-on-instance.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+set -eu
+
+JOB_UPDATE_STAGING="8d452c26-5a39-4162-b3b8-4da5efdf0b76"
```

Comment:
is it always the same? This is a global token for the .rock env?

Reply:
Always the same for the `[cozy.rocks - Staging] Install app on instance` job. 
We'll need another key if we want to run other job.
Do you want me to put an envirronement var in travis ?

Reply:
Yep, it's the job's ID, never changes.

Reply:
Does staging also contains `.red` domain for instance? 

This is just a question to know if we will need to work on it later to be able to deploy on other "env"

Reply:
I'm not sure, let me check with infra.
Anyway if we want to deploy to INT ie, we just need to change (or add) this job, like they did in the old cozy-bank script (https://github.com/cozy/cozy-banks/commit/aa36466d5fc870a8cd659cb32160546bc7ad9e04)

Reply:
`.red` is on INT, not stagging, so we'll need another token 
rundeck jobs should run on `.red` but Seb told me nobody uses it yet!

Reply:
Thanks!